core/src/lib.rs

```diff
@@ -229,6 +235,178 @@ impl SQLValueType for ValueType {
     }
 }
 
+trait EnumSetExtensions<T: enum_set::CLike + Clone> {
```

Comment:
I expect we'll use them whenever we use an `EnumSet`; they're not related to `ValueType`, but they are useful utilities.